src/clojure/com/interrupt/ibgateway/component/figwheel/repl_server.clj

```diff
@@ -0,0 +1,19 @@
+(ns com.interrupt.ibgateway.component.figwheel.repl-server
+  (:require [environ.core :refer [env]]
+            [mount.core :refer [defstate] :as mount]
+            [clojure.tools.nrepl.server :refer [start-server stop-server] :as nrepl]
+            [cider.nrepl :refer [cider-middleware]]
+            [refactor-nrepl.middleware :refer [wrap-refactor]]))
+
+
+(def host "0.0.0.0")
```

Comment:
Let's make `host` and `env` env vars instead of hard-coding them.

Reply:
Agreed. That was the first thing I tried. The repl server was blowing up on those values. Don't know why.